datahub-web-react/src/app/identity/group/GroupList.tsx

```diff
@@ -66,7 +66,7 @@ export const GroupList = () => {
     };
 
     const handleDelete = (urn: string) => {
-        removeGroupFromListGroupsCache(urn, client);
+        removeGroupFromListGroupsCache(urn, client, page, pageSize);
```

Comment:
nice thanks for the fix. We should check all multi-page things (note to myself too)